.github/workflows/reusable-e2e.yml

```diff
@@ -392,8 +393,20 @@ jobs:
         env:
           SEGMENT_DOWNLOAD_TIMEOUT_MINS: 1
 
+      - name: Restore node_modules cache
+        uses: actions/cache@v3
+        id: restore-cache
+        with:
+          path: |
+            ./node_modules
+            **/node_modules
+          key: ${{ runner.os }}-nodemodules-${{ inputs.commit }}
+        env:
+          SEGMENT_DOWNLOAD_TIMEOUT_MINS: 3
+
       - name: Install packages
-        run: yarn install
+        if: steps.restore-cache.outputs.cache-hit != 'true' || steps.restore-cypress-cache.outputs.cache-hit != 'true'
+        uses: ./.github/actions/install-with-retries
```

Comment:
Would it be an issue here without providing any inputs? I notice that by default we will skip install cypress binary

Reply:
You're right, thanks for the catch.

I've added 28bc123b26851a2f76a0c5050660ae11223a680b to address this.